src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,129 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
+
 public class ArrayList<T> implements List<T> {
 
+    private static final int INITIAL_CAPACITY = 1;
```

Comment:
```suggestion
    private static final int INITIAL_CAPACITY = 10;
```
let's make capacity same as in original Arraylist

Reply:
Sure, that's was for testing purposes :)